tests/integration/synthetics/cli.run.test.js

```diff
@@ -213,7 +213,8 @@ describe('cli.run()', () => {
 
     const useCases = [
       { subject: 'bare', args: [] },
-      { subject: 'flatten-components', args: ['--flatten-components'] }
+      { subject: 'flatten-components', args: ['--flatten-components'] },
+      { subject: 'add-license-text', args: ['--add-license-text'] }
```

Comment:
Why did you choose to add a new set of demos? Do we actually need this for some purpose? Why not write an own proper setup?

Reply:
I have chosen the cli-run (describe('cli.run()', ...)) tests to have an e2e test. 
There I have chosen the area where other parameters like --flatten-components were tested.
I didn't get it, that this creates a new set of demos. Can you assist me to find a proper setup to test the --add-license-test parameter? I guess I need some support to find there a good solution.

Reply:
understandable. thanks for clarification <3